versioned_docs/version-3.1.x/security-tls-transport.md

```diff
@@ -237,8 +237,8 @@ servicePortTls=6651
 webServicePortTls=8081
 
 # configure certificates for clients to connect proxy
-tlsCertificateFilePath=/path/to/broker.cert.pem
```

Comment:
Please add the steps as shown below to generate a certificate for the proxy

![image](https://github.com/apache/pulsar-site/assets/1907867/cb92325e-39f6-4964-bb72-210fd3cc1be7)

Reply:
@Technoboy- any updates on this comment?

Reply:
@tuteng please file a separate issue for improvements